token-transfer-server/src/scripts/insert_ogv_matching_grants.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const BigNumber = require('bignumber.js')
+const Logger = require('logplease')
+const moment = require('moment')
+
+const db = require('../models')
+
+const { vestingSchedule } = require('../lib/vesting')
+
+Logger.setLogLevel(process.env.LOG_LEVEL || 'INFO')
+const logger = Logger.create('insert_ogv_airdrop_grants', {
+  showTimestamp: false
+})
+
+const SNAPSHOT_DATE = '2022-7-12'
+
+async function run() {
+  const users = await db.User.findAll({ where: { id: 693 } })
+  for (const user of users) {
+    const grants = await db.Grant.findAll({ where: { userId: user.id } })
```

Comment:
nit: Could actually use `Grant` model in `includes` attribute in the `User.findAll` query above instead